src/inc/msquic.h

```diff
@@ -1657,6 +1713,8 @@ typedef struct QUIC_API_TABLE {
     QUIC_STREAM_PROVIDE_RECEIVE_BUFFERS_FN
                                         StreamProvideReceiveBuffers; // Available from v2.5
 #endif
+    QUIC_CONNECTION_POOL_API_OPEN_FN     ConnectionPoolApiOpen; // Available from v2.5
```

Comment:
Please put it in `QUIC_API_ENABLE_PREVIEW_FEATURES`, make sure it aligns with the other names and align the comment too.